README.md

```diff
@@ -105,22 +107,43 @@ $ npm run check
 
 # Regression
 
-The `regression/run-regression.ts` script can be used to run regression on a set of repos. It takes the following arguments:
+The `regression/cli.ts` script can be used to run regression on a set of repos. It's default command, `run` supports the following options:
+
+```
+Options:
+  -a, --a <version>      Baseline version of SUSHI. Can be an NPM version number or tag, "gh:branch" to use a GitHub branch, or "local" to use the local code with
+                         ts-node. (default: "gh:master")
+  -b, --b <version>      Vesion of SUSHI under test. Can be an NPM version number or tag, "gh:branch" to use a GitHub branch, or "local" to use the local code with
+                         ts-node. (default: "local")
+  -l, --lookback <days>  The number of days to lookback in FSHFinder repositories (based on last updated date).
+  -c, --count <number>   The maximum number of FSHFinder repositories to test (most recent first).
+  -r, --repo <repos...>  One or more repos to test, each specified as a Github {org}/{repo}#{branch} (e.g., HL7/fhir-mCODE-ig#master). This option is not
+                         compatible with the lookback, count, or file options.
+  -f, --file <file>      A text file for which each line is a GitHub {org}/{repo}#{branch} to test (e.g., HL7/fhir-mCODE-ig#master). This is mostly used for legacy
+                         purposes and is not compatible with the lookback, count, or repo arguments.
+  -o, --output <folder>  The folder to write regression data to (default: "regression/output")
+  -h, --help             display help for command
+```
+
+You can run it via ts-node:
 
-- `repoFile`: A text file for which each line is a GitHub `{org}/{repo}#{branch}` to run regression on (e.g.,
-  `HL7/fhir-mCODE-ig#master`). Comment out lines using a leading `#`. _(default: regression/repos-select.txt)_
-- `version1`: The base version of SUSHI to use. Can be an NPM version number or tag, `gh:{branch}` to use a GitHub branch,
-  or `local` to use the local code with `ts-node`. _(default: gh:master)_
-- `version2`: The version of SUSHI under test. Can be an NPM version number or tag, `gh:{branch}` to use a GitHub branch,
-  or `local` to use the local code with `ts-node`. _(default: local)_
+```sh
+$ ts-node regression/cli.ts -a 3.0.0 -b local -c 50
```

Comment:
I feel like 95% of the time, we just want to run it, so I kind of like it as the default since I'm lazy. But I can be convinced otherwise if you and/or @mint-thompson feel strongly about it.

Reply:
Oh! I misunderstood! I thought you were suggesting we remove it as a default. But now I see you're just talking about how we document it in the README. Oops! Yeah, I agree, let's go ahead and put in the `run` to be more explicit.

Reply:
Oh lol you almost had me convinced on that last one that we should leave the documentation using the default. But yes, I definitely agree `run` should be default, and if you took my suggestion on the documentation, then that works for me!